vector/src/main/java/im/vector/app/features/onboarding/OnboardingViewModel.kt

```diff
@@ -118,6 +118,33 @@ class OnboardingViewModel @AssistedInject constructor(
         }
     }
 
+    private fun observeQrCodeLoginCapability(homeServerUrl: String) = viewModelScope.launch {
```

Comment:
Why is method named with `observe` prefix? It seems there is no observation pattern. Here, I would rather name it `checkQrCodeLoginCapability` or `refreshQrCodeLoginCapability`.

Reply:
Done.